shell/driver.postgresql/db-parse

```diff
@@ -21,7 +21,11 @@ echo "export DBHOST DBPORT DBUSER DBPASSWORD DBNAME"
 
 # more variables for Scala application.conf
 echo "DEEPDIVE_JDBC_DRIVER=org.postgresql.Driver"
-echo "DEEPDIVE_JDBC_URL=jdbc:postgresql://$host${port:+:$port}/$dbname"
+if [ ${ssl} = "true" ]; then
+    echo "DEEPDIVE_JDBC_URL=jdbc:postgresql://$host${port:+:$port}/$dbname?ssl=true\&sslfactory=org.postgresql.ssl.NonValidatingFactory"
+else
+    echo "DEEPDIVE_JDBC_URL=jdbc:postgresql://$host${port:+:$port}/$dbname"
+fi
```

Comment:
Sorry, I don't understand 'I think the &sslfactory=org.postgresql.ssl.NonValidatingFactory option should be moved to documentation.'. 
Yes,  `&sslfactory=org.postgresql.ssl.NonValidatingFactory` is for getting self-signed cert, and we add it in to `DEEPDIVE_JDBC_URL` for that purpose.